cli/src/commands/config.ts

```diff
@@ -0,0 +1,39 @@
+import { PublicApiUrl } from '@massalabs/massa-web3'
+import { OptionValues } from 'commander'
+import { readFileSync } from 'fs'
+
+export const DEFAULT_CHUNK_SIZE = 64000
+const DEFAULT_NODE_URL = PublicApiUrl.Buildnet
+
+interface Config {
+  wallet_password: string
+  wallet_path: string
+  node_url: string
+  chunk_size: number
+  secret_key: string
+}
+
+export function parseConfigFile(filePath: string): Config {
+  const fileContent = readFileSync(filePath, 'utf-8')
+  try {
+    return JSON.parse(fileContent)
+  } catch (error) {
+    throw new Error(`Failed to parse file: ${error}`)
+  }
+}
```

Comment:
CI is failing with:
```
/home/runner/work/DeWeb/DeWeb/cli/bin/commands/utils.js:42
        throw new Error('node_url is not defined. Please use --node_url to set one');
```

I guess it's because of the Config interface. As everything is a string, if the key is not found, `JSON.parse` might put the default value for the type, so for a string, an empty string, which is not undefined, so during the merge, this should be the taken value. A solution to that would be to change the `string` types to:
```suggestion
interface Config {
  wallet_password?: string
  wallet_path?: string
  node_url?: string
  chunk_size: number
  secret_key?: string
}

export function parseConfigFile(filePath: string): Config {
  const fileContent = readFileSync(filePath, 'utf-8')
  try {
    return JSON.parse(fileContent)
  } catch (error) {
    throw new Error(`Failed to parse file: ${error}`)
  }
}
```

This change should allow `JSON.parse` to put an `undefined` if the key is not present in the config file.

Reply:
Well in fact, we don't even go into this function in the case of the CI as we don't have a config file there. 

So the issue is when we put the default values. ATM, we put them in the `mergeConfigAndOptions` function, which is only called if there is a config file. Maybe we should move default values handling from `mergeConfigAndOptions` to a function that is called right before `program.parse()` that will check if some default values are missing, and add them if so ?

Reply:
This would come after the mergeFunction... it could be an elegant solution to keep default at a top level!

Reply:
I pushed a solution => not exactly what you suggested because if we call a set default option before the parse we would make an extra call if there is a configFile. I just added a setDefaultValues in an else statement after the initial if